primitives/src/trie/mpt.rs

```diff
@@ -352,25 +370,11 @@ impl MptNode {
     ///
     /// Nibbles are half-bytes, and in the context of the MPT, they represent parts of
     /// keys.
+    #[inline]
     pub fn nibs(&self) -> Vec<u8> {
         match &self.data {
             MptNodeData::Null | MptNodeData::Branch(_) | MptNodeData::Digest(_) => vec![],
-            MptNodeData::Leaf(prefix, _) | MptNodeData::Extension(prefix, _) => {
```

Comment:
Do we have bug in mpt side?? if not, maybe we can make this PR cleaner and make another PR for mpt.

Reply:
These codes are the newest from upstream